ddtrace/span.py

```diff
@@ -486,14 +487,19 @@ def pprint(self):
             ("resource", self.resource),
             ("type", self.span_type),
             ("start", self.start),
-            ("end", "" if not self.duration else self.start + self.duration),
-            ("duration", "%fs" % (self.duration or 0)),
+            ("end", None if not self.duration else self.start + self.duration),
+            ("duration", self.duration),
             ("error", self.error),
-            ("tags", ""),
+            ("tags", dict(sorted(self.meta.items()))),
+            ("metrics", dict(sorted(self.metrics.items()))),
         ]
-
-        lines.extend((" ", "%r:%s" % kv) for kv in sorted(self.meta.items()))
-        return "\n".join("%10s %s" % line for line in lines)
+        pprinted = []
+        for (k, v) in data:
+            buff = StringIO()
+            # use a large column width to keep pprint output on one line
+            pprint.pprint(v, width=1024 ** 2, stream=buff)
```

Comment:
would `pformat` be better?

https://docs.python.org/2.7/library/pprint.html#pprint.pformat